src/value_classes/value_id.rs

```diff
@@ -397,7 +460,7 @@ use std::cmp::{self, Ordering};
 
 impl cmp::PartialEq for ValueID {
     fn eq(&self, other: &ValueID) -> bool {
-        unsafe { extern_value_id::value_id_eq(self.ptr, other.ptr) }
+        self.home_id == other.home_id && self.id == other.id
     }
```

Comment:
Possibly. I noticed that PartialOrd calls self.eq and I wasn't sure if that would be implemented if we dreived. I'll try it.